common-lib/src/main/scala/com/gu/mediaservice/lib/aws/DynamoDB.scala

```diff
@@ -128,6 +128,14 @@ class DynamoDB(credentials: AWSCredentials, region: Region, tableName: String) {
         new ValueMap().withMap(":value", valueMapWithNullForEmptyString(value))
     )
 
+  def jsonPatch(id: String, key: String, value: Map[String, String])
```

Comment:
@theefer added this so that we can patch the JSON map.

This will come in handy as we could potentially split the metadata form into single inputs not doing all the merging / validation on the client.